react-native-aztec/ios/RNTAztecView/RCTAztecView.swift

```diff
@@ -29,10 +29,38 @@ class RCTAztecView: Aztec.TextView {
 
     private var previousContentSize: CGSize = .zero
 
+    var textHorizontalInset: CGFloat {
+        return contentInset.left + textContainerInset.left + textContainer.lineFragmentPadding
+    }
+
+    var isRTLLayout: Bool {
```

Comment:
This is pretty small, but considering this is a property of `RCTAztecView`, a more precise naming for this property could be `hasRTLLayout`.